can't compile rust crate for on macOS

it seems that after #56 this package can no longer build on arm64 architectures on macos (haven't tried amd46)
the current error I'm seeing when building is
```
Undefined symbols for architecture arm64:
  "_tree_sitter_elixir", referenced from:
      tree_sitter_elixir::language::hdfb5f5f23586403c in libtree_sitter_elixir-37811105d665b052.rlib(tree_sitter_elixir-37811105d665b052.tree_sitter_elixir.e516692c-cgu.0.rcgu.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

clang --version output
```
Apple clang version 14.0.3 (clang-1403.0.22.14.1)
Target: arm64-apple-darwin22.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

rust toolchain
```
[toolchain]
channel = "1.70"
components = [ "rustfmt" ]
targets = [ "aarch64-apple-darwin" ]
```

cargo.toml
```
[dependencies]
tree-sitter-elixir = { git = "https://github.com/elixir-lang/tree-sitter-elixir", rev = "59a2ba5c0d768f8a7a6de8caa7f523d03ec6b647" }
```

tried on amd64 and it also doesn't seem to work
```
Undefined symbols for architecture x86_64:
            "_tree_sitter_elixir", referenced from:
                tree_sitter_elixir::language::hcb8433d13b3129e1 in libtree_sitter_elixir-fecd53fdffcbf903.rlib(tree_sitter_elixir-fecd53fdffcbf903.tree_sitter_elixir.a5745211-cgu.0.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

```
Apple clang version 14.0.3 (clang-1403.0.22.14.1)
Target: x86_64-apple-darwin22.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

--- a/bindings/rust/build.rs
+++ b/bindings/rust/build.rs
@@ -9,8 +9,11 @@ fn main() {
         .flag_if_supported("-Wno-trigraphs");
     let parser_path = src_dir.join("parser.c");
     c_config.file(&parser_path);
+    println!("cargo:rerun-if-changed={}", parser_path.to_str().unwrap());
 
     let scanner_path = src_dir.join("scanner.c");
     c_config.file(&scanner_path);
     println!("cargo:rerun-if-changed={}", scanner_path.to_str().unwrap());
+
+    c_config.compile("parser-scanner");
 }

